code/extensions/Lumberjack.php

```diff
@@ -38,7 +39,7 @@ public function getExcludedSiteTreeClassNames()
      */
     public function updateCMSFields(FieldList $fields)
     {
-        $excluded = $this->owner->getExcludedSiteTreeClassNames();
+        $excluded = $this->owner->getExcludedSiteTreeClassNames(true);
```

Comment:
I think it might be clearer to move this into a new method, rather than adding a flag to an existing one. Maybe `getLumberjackGridfieldClassNames` in the positive sense?